niworkflows/utils/bids.py

```diff
@@ -223,19 +237,10 @@ def collect_data(
         'return_type': 'file',
         'subject': participant_label,
         'extension': ['.nii', '.nii.gz'],
-        'session': session_id,
+        'session': session_id or Query.OPTIONAL,
```

Comment:
This changes the meaning of None from Query.NONE, which diverges from pybids semantics. I think that's an okay change, but we should check that we never use None in the way now.

Reply:
You are right, didn't think of it. Considering that single session and multisession datasets cannot be mixed up in BIDS, I agree this should be okay.